src/core/platform/f3xx/stm32f302x8.cpp

```diff
@@ -20,13 +20,13 @@ void stm32f3xx_init() {
     RCC_OscInitStruct.HSICalibrationValue = RCC_HSICALIBRATION_DEFAULT;
     RCC_OscInitStruct.PLL.PLLState        = RCC_PLL_ON;
     RCC_OscInitStruct.PLL.PLLSource       = RCC_PLLSOURCE_HSI;
-    RCC_OscInitStruct.PLL.PLLMUL          = RCC_PLL_MUL4; // PLL CLK at 8 * 4 = 32 MHz
+    RCC_OscInitStruct.PLL.PLLMUL          = RCC_PLL_MUL8; // PLL CLK at 8 / 2 * 8 = 32 MHz
     HAL_RCC_OscConfig(&RCC_OscInitStruct);
 
     /** Initializes the CPU, AHB and APB buses clocks
      */
```

Comment:
Unimportant change but just make this a 1 line comment instead of having the `*/` on the next line, no?